build/zlib-1.2.8/contrib/vstudio/vc14/zlibstat.vcxproj

```diff
@@ -212,26 +185,27 @@
       <WarningLevel>Level3</WarningLevel>
       <SuppressStartupBanner>true</SuppressStartupBanner>
       <DebugInformationFormat>OldStyle</DebugInformationFormat>
-      <RuntimeLibrary>MultiThreadedDebugDLL</RuntimeLibrary>
+      <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
       <ProgramDataBaseFileName> $(OutDir)$(TargetName).pdb</ProgramDataBaseFileName>
     </ClCompile>
     <ResourceCompile>
       <Culture>0x040c</Culture>
     </ResourceCompile>
     <Lib>
       <AdditionalOptions>/MACHINE:X86 /NODEFAULTLIB %(AdditionalOptions)</AdditionalOptions>
-      <OutputFile>$(OutDir)zlibwapi.lib</OutputFile>
+      <OutputFile>$(OutDir)zlibstat.lib</OutputFile>
```

Comment:
Could you keep 'wapi' in the name? Maye zlibwapi-static.lib ? 

The Internet is plagued with copies of zlib built with an unmarked calling convention :).